Formula/react-native-cli.rb

```diff
@@ -1,7 +1,7 @@
 require "language/node"
 
 class ReactNativeCli < Formula
-  desc "The React Native CLI tools"
+  desc "Tools for creating native apps for Android and iOS"
```

Comment:
```suggestion
  desc "Tools for creating native React apps for Android and iOS"
```

Alternatively:

```suggestion
  desc "Tools for creating native apps with React for Android and iOS"
```

Not super familiar with React so I'm not sure what the best description would be

Reply:
Kinda redundant when React is already part of the name.

Reply:
I feel less strongly about this one than I do about `opencsg` but I still think it doesn't hurt to clarify. To be fair, adding the words, in this case, makes it a little harder to read. I wouldn't be upset if this wasn't included.